flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/hybrid/HsSpillingInfoProvider.java

```diff
@@ -79,4 +79,24 @@ enum ConsumeStatus {
         /** The buffer is either consumed or not consumed. */
         ALL
     }
+
+    /** This class represents a pair of {@link ConsumeStatus} and consumer id. */
+    class ConsumeStatusWithId {
+        public static final ConsumeStatusWithId ALL_CONSUME_STATUS =
```

Comment:
I'd suggest the name `ALL_ALL` or `ANY_ANY`, following the pattern `ConsumeStatus_ConsumerID`. If more special values are needed in future, we can follow the same pattern.